core/src/main/java/org/geysermc/geyser/translator/collision/BlockCollision.java

```diff
@@ -166,4 +166,15 @@ public double computeCollisionOffset(double x, double y, double z, BoundingBox b
         }
         return offset;
     }
+
+    public boolean isBelow(double y, BoundingBox boundingBox) {
```

Comment:
Collision code is already confuzzling - a Javadoc here might be helpful, but no pressure if not.